MyHeartCounts/Onboarding/ComprehensionScreening.swift

```diff
@@ -71,3 +71,11 @@ private struct Question: ScreeningComponent {
         data.comprehension[keyPath: storage] == true
     }
 }
+
+
+#Preview {
+    ManagedNavigationStack {
+        ComprehensionScreening()
+    }
+        .environment(OnboardingDataCollection())
```

Comment:
just out of curiosity: does your Xcode default to indenting modifiers in scenarios like this? or do you add the indent level by hand?